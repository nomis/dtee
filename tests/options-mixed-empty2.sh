#!/bin/bash
set -vx
. "$(dirname "$0")"/util/common.sh

# Empty filename followed by empty command with empty argument
run_test_once "--debug-options" "-o" "" "" ""
RET=$?

echo RET $RET
if [ $RET -eq 0 ]; then
	exit 0
fi
exit 1
