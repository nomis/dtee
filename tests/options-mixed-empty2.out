-=
-=
debug-options=1
out-append=
